docs/dev_tools/third_party.md

```diff
@@ -12,6 +12,7 @@ If you'd like to develop on mbed OS with a third party tool, or migrate to one,
     - GNU ARM Embedded Toolchain.
     - ARM Compiler 5.
     - IAR ARM Compiler.
+- GNU ARM Eclipse (managed CDT projects), using GNU ARM Embedded Toolchain.
```

Comment:
Please define "CDT" this first time it's used for the readers who aren't as familiar with it.